src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
 spring.sql.init.mode=always
+spring.datasource.url=jdbc:postgresql://localhost:${POSTGRES_PORT}/${POSTGRES_DB}
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=${POSTGRES_USER}
+spring.datasource.password=${POSTGRES_PASSWORD}
+spring.datasource.hikari.maximum-pool-size=10
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
nit:
Same comment about commit message length. Add properties to properties, sounds like a duplication.

Also, newline at the end is missing, although I'm not sure if this thing can have one